backend/util/GameUtil.js

```diff
@@ -2,13 +2,13 @@
 class GameUtil {
 
     static ROOMS_ID = [];
+    static GUEST_ID = [];
```

Comment:
It is an array, shouldn't it be called "GUEST_IDS" ?

Reply:
You're right